import.sql

```diff
@@ -1,6 +1,6 @@
 # comment this out if your server has a different table/location for phone_number
 # or if it already exists in the users table
-# ALTER TABLE `users` ADD COLUMN `phone_number` VARCHAR(20) DEFAULT NULL;
+ALTER TABLE `users` ADD COLUMN `phone_number` VARCHAR(20) DEFAULT NULL;
```

Comment:
We have to check if the table exists.

Reply:
> We have to check if the table exists.

Isnt those checks dependent on MYSQL versions?

Reply:
I don't think 'if exists' is

Reply:
I don't think 'if exists' is